lib/Cake/Test/Case/Utility/HashTest.php

```diff
@@ -1361,6 +1361,38 @@ public function testSortNaturalFallbackToRegular() {
 	}
 
 /**
+ * Test sort() with locale option.
+ *
+ * @return void
+ */
+	public function testSortLocale() {
+		// get the current locale
+		$oldLocale = setlocale(LC_COLLATE, '0');
+
+		// the de_DE.utf8 locale must be installed on the system where the test is performed
+		setlocale(LC_COLLATE, 'de_DE.utf8');
```

Comment:
@dereuromark Our 3.x testsuite runs on travis' "trusty" VMs which probably have the locale already.